Source/WebCore/platform/graphics/Image.h

```diff
@@ -197,8 +205,8 @@ class Image : public RefCountedAndCanMakeWeakPtr<Image> {
     WeakPtr<ImageObserver> m_imageObserver;
     std::unique_ptr<ImageAdapter> m_adapter;
 
-    // A value of true or false will override the default Page::imageAnimationEnabled state.
-    std::optional<bool> m_allowsAnimation { std::nullopt };
+    // This will override the default Page::imageAnimationEnabled state.
```

Comment:
I think one could argue that the enum class has made this self-documenting, thus making the comment redundant? Fine to leave it if you'd prefer though.